src/llvm-multiversioning.cpp

```diff
@@ -1025,6 +1025,9 @@ bool MultiVersioning::runOnModule(Module &M)
     //     * Cloned function -> Original function (add as we clone functions)
     //     * Original function -> Base function (target specific and updated by LLVM)
     //     * ID -> relocation slots (const).
+    if (M.getName() == "sysimage")
+        return false;
```

Comment:
Why do you turn of multi-versioning from the sysimage?

Reply:
"sysimage" doesn't have any text in it, only data